### ISSUE TITLE
Error since update to 6.4.0

### ISSUE THREAD

**Issue by t3easy** [AOEpeople/crawler#352]
Since the update from 6.3.2 with PHP 5.6 to 6.4.0 with PHP 7.2 we receive the following error:
```
 [ Error ]                                                                                
 Call to a member function push() on null                                                 
 thrown in file typo3/sysext/frontend/Classes/Controller/TypoScriptFrontendController.php 
 in line 1287                                                                             
                                                                                          

Exception trace:
#0 TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController::determineId()
   typo3conf/ext/crawler/Classes/Controller/CrawlerController.php:2709
#1 AOE\Crawler\Controller\CrawlerController::initTSFE()
   typo3conf/ext/crawler/Classes/Controller/CrawlerController.php:1349
#2 AOE\Crawler\Controller\CrawlerController::readUrl()
   typo3conf/ext/crawler/Classes/Controller/CrawlerController.php:2367
#3 AOE\Crawler\Controller\CrawlerController::CLI_run()
   typo3conf/ext/crawler/Classes/Controller/CrawlerController.php:2092
#4 AOE\Crawler\Controller\CrawlerController::CLI_main()
   typo3conf/ext/crawler/Classes/Task/CrawlerTask.php:67
#5 AOE\Crawler\Task\CrawlerTask::execute()
   typo3/sysext/scheduler/Classes/Scheduler.php:158
#6 TYPO3\CMS\Scheduler\Scheduler::executeTask()
   vendor/helhum/typo3-console/Classes/Command/SchedulerCommandController.php:70
#7 Helhum\Typo3Console\Command\SchedulerCommandController::executeScheduledTasks()
   vendor/helhum/typo3-console/Classes/Command/SchedulerCommandController.php:52
#8 Helhum\Typo3Console\Command\SchedulerCommandController::runCommand()
#9 call_user_func_array()
   vendor/helhum/typo3-console/Classes/Mvc/Controller/CommandController.php:252
#10 Helhum\Typo3Console\Mvc\Controller\CommandController::callCommandMethod()
   vendor/helhum/typo3-console/Classes/Mvc/Controller/CommandController.php:137
#11 Helhum\Typo3Console\Mvc\Controller\CommandController::processRequest()
   typo3/sysext/extbase/Classes/Mvc/Dispatcher.php:86
#12 TYPO3\CMS\Extbase\Mvc\Dispatcher::dispatch()
   vendor/helhum/typo3-console/Classes/Mvc/Cli/RequestHandler.php:80
#13 Helhum\Typo3Console\Mvc\Cli\RequestHandler::handleRequest()
   vendor/helhum/typo3-console/Classes/Core/ConsoleBootstrap.php:111
#14 Helhum\Typo3Console\Core\ConsoleBootstrap::run()
   vendor/helhum/typo3-console/Scripts/typo3cms.php:60
#15 {closure}()
   vendor/helhum/typo3-console/Scripts/typo3cms.php:61
#16 require()
   vendor/helhum/typo3-console/Scripts/typo3cms:4
```

Environment:
- crawler 6.4.0
- TYPO3 7.6
- PHP 7.2
- scheduler
- TYPO3 Console 4.9.6

**Comment by tomasnorre** [comment]
@t3easy 

How to reproduce this? When does this happen?

**Comment by t3easy** [comment]
This happens during the "Crawler Run (crawler)" scheduler job.

**Comment by t3easy** [comment]
Even if I start the job in the scheduler backend.

**Comment by tomasnorre** [comment]
Thanks.. Will check it right away..

**Comment by t3easy** [comment]
you are not building $GLOBALS['TT']; anymore

**Comment by t3easy** [comment]
https://github.com/AOEpeople/crawler/compare/6.3.2...6.4.0#diff-481209dc0ee6be420867b4ac30992471R2702

**Comment by tomasnorre** [comment]
True.. Cannot open your link. 

The $GLOBALS['TT'] is deprecated for TYPO3 v8, so we dropped it, and thought it would be enough as we do it here.

https://github.com/AOEpeople/crawler/blob/master/Classes/Controller/CrawlerController.php#L2701

**Comment by tomasnorre** [comment]
Resolved in: 
https://github.com/AOEpeople/crawler/commit/dc738f984067e6c34ac90cc04e7884c296c3c9a7

Released 6.4.1
